app/src/main/java/com/zulip/android/networking/ZulipInterceptor.java

```diff
@@ -0,0 +1,37 @@
+package com.zulip.android.networking;
+
+import android.util.Base64;
+
+import com.zulip.android.ZulipApp;
+
+import java.io.IOException;
+
+import okhttp3.Interceptor;
+import okhttp3.Request;
+import okhttp3.Response;
+
+/**
+ * Created by patrykpoborca on 8/25/16.
+ */
```

Comment:
Can you remove this authorship block? the Zulip project uses git as a record of authorship.